docs/local-development.md

```diff
@@ -0,0 +1,29 @@
+# Local Development
+
+## Requirements
+
+- docker v20.10. Required for host.docker.internal on UNIX systems
```

Comment:
`docker-compose` is missing in the list

Reply:
What's the difference between this list and the list in `how-to-contribute.yml`?

Reply:
how-to-contribute.md gives instruction on how to build a permission-manager image and deploy it to kubernetes cluster.

It is different from the local development one

It should be given better names to the files IMO